uwblueprint/community-fridge-kw
# feat: update recurring schedules

## Brief description. What is this change? 

[Update recurring schedules frontend](https://www.notion.so/uwblueprintexecs/Update-recurring-schedules-frontend-a7357789bcd84a74b0f325bdc79905df)
[Update recurring schedules backend](https://www.notion.so/uwblueprintexecs/Update-recurring-schedules-backend-67cc9625196f48209887960a8635e304)
[volunteer information fields edits](https://www.notion.so/uwblueprintexecs/volunteer-information-fields-not-updated-properly-when-edited-64796b0c86c64f669dcabf284718aea5)





## Implementation description. How did you make this change?
* Frontend changes to modularize recurringDonationModal to work for deletion and when modifying
* Backend changes to update recurring donations based on recurring donation id
* Error and success toast



## Steps to test
1. Edit recurring and single donations from all three screens (date and time, donation info, volunteer info) to make sure everything is being edited properly. As discussed and decided, you cannot edit recurring donations on date and time screen.
2. Check if modals and toasts are shown properly
3. Come back to the donation or check on pgadmin to see if values updated properly




## Checklist
